iam/x509-identity-mgmt/js/src/express/routes/throwAwayRoutes.js

```diff
@@ -23,21 +28,31 @@ module.exports = ({ mountPoint, schemaValidator, errorTemplate }) => {
         method: 'post',
         middleware: [
           validateRegOrGenCert(),
-          async (req, res) => {
-            let result = null;
+          async (req, res, next) => {
             if (req.body.csr) {
               const csr = sanitizeParams.sanitizeLineBreaks(req.body.csr);
               const belongsTo = req.body.belongsTo || {};
 
               const certService = req.scope.resolve(CERT_SERVICE);
-              result = await certService.throwAwayCertificate({
+              res.result = await certService.throwAwayCertificate({
                 csr, belongsTo,
               });
+              next();
             } else {
               throw BadRequest('It is necessary to inform the CSR for the certificate to be issued.');
             }
-            res.status(HttpStatus.CREATED).json(result);
           },
+          contentNegotiation(accepts, {
+            'application/x-pem-file': (res) => {
```

Comment:
maybe this could be a function it receives if the parameters `res.result.certificatePem`,` filename` and `HttpStatus.CREATED`, because there seems to be a replication of this behavior

Reply:
but maybe as it is it makes it clearer what's going on

Reply:
done!